app/api/routers/automatic.py

```
@@ -128,3 +132,143 @@ async def cleanup_session(session_id: str):
+                    arg_name = arg_map.get(key)
+                    if arg_name:
+                        if isinstance(value, list):
+                            cmd.extend([arg_name] + value)
+                        elif isinstance(value, bool):
+                            if value:  # Only add boolean flags if True
+                                cmd.append(arg_name)
+                        else:
+                            cmd.extend([arg_name, str(value)])
+
+            # Start the subprocess
+            proc = subprocess.Popen(
```

Ruff (0.14.3)

`subprocess` call: check for execution of untrusted input

(S603)